packages/appstore/src/modules/trading-hub/index.tsx

```diff
@@ -80,17 +97,58 @@ const TradingHub: React.FC = () => {
     );
 
     return (
-        <React.Fragment>
-            <div>
-                <TotalAssets category={tab_account_type} />
-                <ToggleAccountType
-                    accountTypeChange={(event: any) => accountTypeChange(event)}
-                    value={tab_account_type}
-                />
+        <div id='trading-hub' className='trading-hub'>
+            <div className='trading-hub_header'>
+                <div className='trading-hub_header--title'>
+                    <Text weight='bold' size={isMobile ? 'xxs' : 'm'} align='left'>
```

Comment:
I think these changes are not reflected. Text is "xsm" in the test link. 🤔

Reply:
oh, it's because I've missed () after `isMobile`, will fix that